ERCS/erc-draft_wrap_revert.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Wrapping of bubbled up reverts
+description: Handling bubbled up reverts using custom errors with additional context
+author: Daniel Gretzke (@gretzke), Sara Reynolds (@snreynolds), Alice Henshaw (@hensha256), Marko Veniger (marko.veniger@tenderly.co), Hadrien Croubois (@Amxx)
+discussions-to: https://ethereum-magicians.org/t/tbd
```

Comment:
Please create the discussions topic in Eth Magicians.  It just needs to have a link to this PR.

Reply:
I'm trying haha, but I don't see a button to create a new topic. I reached out on telegram to fix the issue

Reply:
![image](https://github.com/user-attachments/assets/8049c73b-cde4-4cbf-83ec-837999f5f9ce)

this is how it's supposed to look like and looks like for a colleague

Reply:
this is how it looks like for me
![image](https://github.com/user-attachments/assets/3af35cbc-848b-478b-a1e5-f5bcfb3c7f7a)

Reply:
my colleague now posted on my behalf, I updated the link in the ERC